allocator/mimalloc-sys/build.rs

```diff
@@ -52,28 +49,6 @@ fn compile_mimalloc(dst: &Path) {
     //build.opt_level(3);
 
     build.compile("mimalloc");
```

Comment:
Since we get some spurious warning spew in the build when this part compiles, I wonder if it's worth including some kind of "no warn" flag here. I think that can be done with `.warnings(false)` (docs at https://docs.rs/cc/latest/cc/struct.Build.html#method.warnings).

Reply:
I'd rather wait on https://github.com/rust-lang/cc-rs/pull/1009

Reply:
It doesn't work the way I thought it worked anyway, so it wouldn't have helped!